static/docs/commands-reference/install.md

```diff
@@ -285,6 +285,6 @@ Data and pipelines are up to date.
 
 After reproducing this pipeline up to the "evaluate" stage, the data files are
 in sync with the code/config files, but we must now commit the changes to the
-Git repository. Looking closely we see that `dvc status` is run again, informing
-us that the data files are synchronized with the `Pipelines are up to date.`
-message.
+Git repository. Looking closely we see that `dvc status` is used again,
+informing us that the data files are synchronized with the
+`Pipelines are up to date.` message.
```

Comment:
please, check the message. I think it's a different one now.

Reply:
True. Updated to `Data and pipelines are up to date.` in 1665812.